test/data/events.json

```diff
@@ -1,336 +1,382 @@
 [
-    {
-        "data": {
-            "annotations": {},
-            "claims": {
-                "expires_human": "never",
-                "issuer": "ACOJJN6WUP4ODD75XEBKKTCCUJJCY5ZKQ56XVKYK4BEJWGVAOOQHZMCW",
-                "name": "HTTP Server",
-                "not_before_human": "immediately",
-                "tags": null,
-                "version": "0.16.0"
-            },
-            "contract_id": "wasmcloud:httpserver",
-            "image_ref": "wasmcloud.azurecr.io/httpserver:0.16.0",
-            "instance_id": "a518251a-01f6-4ba6-b7f0-5d8458bc875e",
-            "link_name": "default",
-            "public_key": "VAG3QITQQ2ODAOWB5TTQSDJ53XK3SHBEIFNK4AYJ5RKAX2UNSCAPHA5M"
-        },
-        "datacontenttype": "application/json",
-        "id": "5613ee03-8645-4ad8-a3aa-3428a33b7e96",
-        "source": "NB6PMW4RGLBP3NAVUVO2IH34VFJFSX7LF7TJOQCDU4GGUGF3P57SZLPX",
-        "specversion": "1.0",
-        "time": "2023-02-14T19:21:09.011785Z",
-        "type": "com.wasmcloud.lattice.provider_started"
+  {
```

Comment:
Note that most of this file isn't actually changed, it's just me adding an additional host heartbeat for the v82 format. Formatting changes other than that.